### DISCUSSION TITLE
Target fields of `searchImage`

### DISCUSSION THREAD

**Discussion by jihoon-seo** [cloud-barista/cb-tumblebug#544] [General]
Target fields of `searchImage`

현재: TB에 등록된 image object 들의 'name' 필드에서 찾습니다.
이 검색 영역을 cspImageName, description, guestOs 등으로도 확장하는 것에 대한 논의가 필요합니다.

**Comment by seokho-son** [comment]
@jihoon-seo 이미지 관리 기능을 완벽하게 제공하는 것은 CB-TB 자체적으로 처리하기 어려움이 있을 것 같습니다.
최소한의 보조기능으로 생각하고
말씀하신 검색의 범위를
cspImageName, description, guestOs 등 영역을 확장하도록 제공해주는 것은 좋을 것 같습니다. (필드를 지정하는 것이 아니라, 단순히 요청한 문자열의 포함 여부만 판단하여 리턴)
